spec/controllers/users/mfa_selection_controller_spec.rb

```diff
@@ -0,0 +1,156 @@
+require 'rails_helper'
+
+describe Users::MfaSelectionController do
+  before do
+    allow(IdentityConfig.store).to receive(:select_multiple_mfa_options).and_return(true)
+    user = build(:user, :signed_up)
+    stub_sign_in(user)
+  end
+  let(:current_sp) { create(:service_provider) }
+
+  describe 'GET index' do
```

Comment:
style, I'd name it like this:
```suggestion
  describe '#index' do
```